protocol/gethservice/broadcaster.go

```diff
@@ -38,7 +45,10 @@ func (b *broadcaster) start(cancel chan struct{}) {
 			for _, out := range subs {
 				out <- item
 			}
+		case <-sub.Err():
+			return
```

Comment:
Let's add a `TODO` comment to handle this error.

Reply:
sorry missed this comment